BlockEntity/BEFruitpress.cs

```diff
@@ -505,55 +575,58 @@ public bool OnBlockInteractStep(float secondsUsed, IPlayer byPlayer, EnumFruitPr
         {
             if (section != EnumFruitPressSection.Screw) return false;
 
-            if (mashStack != null)
-            {
-                RunningAnimation anim = animUtil.animator.GetAnimationState("compress");
-                if (anim != null)
-                {
-                    udpateSqueezeRel(anim);
-                }
-            }
+            if (mashStack != null) updateSqueezeRel(animUtil.animator.GetAnimationState("compress"));
 
-            return CompressAnimActive && secondsUsed < 4f;
+            return CompressAnimActive;
         }
 
 
         public void OnBlockInteractStop(float secondsUsed, IPlayer byPlayer)
         {
-            RunningAnimation anim = animUtil.animator.GetAnimationState("compress");
-            udpateSqueezeRel(anim);
+            updateSqueezeRel(animUtil.animator.GetAnimationState("compress"));
 
             if (!CompressAnimActive) return;
 
-            if (secondsUsed >= 4.8f)

```

Comment:
I noticed that when I removed this and the earlier seconds used conditions the press started to act more predictably. The server animation would flip out if I changed it to run at 0.5f instead of 1f before I removed these conditions, but removing them not only fixed that, it also fixed the issue where you could hold the crank down for 4.8 seconds and then let go of the mouse button, or sometimes just clicking right click fast enough, and it would finish screwing by itself.